charts/kubescape-operator/templates/synchronizer/configmap.yaml

```diff
@@ -192,6 +192,12 @@ data:
             "version": "v1",
             "resource": "servicesscanresults",
             "strategy": "patch"
+          },
+          {
+            "group": "kubescape.io",
+            "version": "v1alpha1",
+            "resource": "operatorcommands",
```

Comment:
this means we're sending them back to backend, is it useful?

Reply:
Yes.
As part of the flow, we send them back to the BE as it reads their status